src/handlers/dialog_handler.pyx

```diff
@@ -0,0 +1,59 @@
+# Copyright (c) 2012 CEF Python, see the Authors file.
```

Comment:
If this is a new file with complete new code then copyright year should be set to 2017